LayoutKitTests/DensityAssertions.swift

```diff
@@ -41,7 +42,12 @@ func AssertEqualDensity(actual: CGFloat, _ expected: [CGFloat: CGFloat], file: S
 
 /// Returns the expectation for the current density.
 private func expectationForCurrentDensity<T>(expected: [CGFloat: T], file: StaticString, line: UInt) -> T? {
-    let scale = UIScreen.mainScreen().scale
+    #if os(iOS)
+        let scale = UIScreen.mainScreen().scale
+    #else
+        let scale: CGFloat = 2.0
```

Comment:
Looks like there's a macOS alternative as well: http://stackoverflow.com/questions/11067066/mac-os-x-best-way-to-do-runtime-check-for-retina-display